Remotion/Core/Development.Analyzers/RMCORE0001_SafeContextAnalyzer.cs

```diff
@@ -14,6 +12,8 @@ namespace Core.Development.Analyzers
   [DiagnosticAnalyzer(LanguageNames.CSharp)]
   public class RMCORE0001_SafeContextAnalyzer : DiagnosticAnalyzer
   {
+    private record SymbolContext (ITypeSymbol? safeContextBoundarySymbol);
```

Comment:
Since the property is public please rename to `SafeContextBoundarySymbol` (capital S)